### DISCUSSION TITLE
Type 'typeof PrismaPlugin' is not assignable to type '"prisma"' in tutorial project

### DISCUSSION THREAD

**Discussion by nebulahh** [prisma/prisma#18509] [Q&A] [answered]
hello, i am following the tutorial on https://www.prisma.io/blog/e2e-type-safety-graphql-react-3-fbV2ZVIGWg#create-a-reusable-instance-of-prisma-client. after creating a reusable instance, i get the error ```Type 'typeof PrismaPlugin' is not assignable to type '"prisma"'.```. the error went away after removing the plugin property but is that how can i fix it

this is the builder file

```js
import SchemaBuilder from "@pothos/core";
import { DateResolver } from "graphql-scalars";
import { PrismaPlugin } from "@pothos/plugin-prisma";
import type PrismaTypes from "@pothos/plugin-prisma/generated";
import { prisma } from './db';

export const builder = new SchemaBuilder<{
  Scalars: {
    Date: { Input: Date; Output: Date};
  };
  PrismaTypes: PrismaTypes;
}>({
  plugins: [PrismaPlugin], // Type 'typeof PrismaPlugin' is not assignable to type '"prisma"'.
  prisma: {
    client: prisma,
  }
});

builder.addScalarType('Date',  DateResolver, {});

builder.queryType({})
```

**Comment by ludralph** [comment]
Hi @nebulahh 👋 

Can you please share a minimal reproduction of this issue in a repository so i can investigate this?

**Reply by nebulahh** [reply]
here is the repo - https://github.com/nebulahh/graphqlApiFromPrisma

**Reply by ludralph** [reply, marked as answer]
Hi @nebulahh 

Apologies for the late response here. This must have slipped through the cracks. 

To resolve your issue, You will need to import `PrismaPlugin` like this
```js
import PrismaPlugin from "@pothos/plugin-prisma";
```
You can look at this section of the [Pothos documentation](https://pothos-graphql.dev/docs/plugins/prisma-utils#setup:~:text=import%20PrismaPlugin%20from%20%27%40pothos/plugin%2Dprisma%27%3B)

**Reply by ludralph** [reply]
Hi @nebulahh 

Do you have any more questions about this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
